### PR TITLE
Add page support for product listing

### PR DESCRIPTION
**Pull request by dloranc**

### CODE REVIEW

File: pages/kategoria/_category/_subcategory/index.vue [hunk 1]
```diff
@@ -5,9 +5,12 @@
 </template>
 
 <script>
-import API from '~/middleware/api.js'
+import { createNamespacedHelpers } from 'vuex'
+import { getProductsFromCategory } from '~/models/category'
 import ProductListing from '~/components/Product/Listing/ProductListing.vue'
 
+const { mapActions } = createNamespacedHelpers('shop')
```
Comment (jan-gajdowicz) [inline pages/kategoria/_category/_subcategory/index.vue:12]: Please get rid of this, and use regular `this.$store.dispatch`

File: pages/kategoria/_category/_subcategory/index.vue [hunk 2]
```diff
@@ -44,29 +41,30 @@ export default {
 
 			this.pageIndex += 1
 
-			this.$store.dispatch('shop/setLoader', true)
+			this.setLoader(true)
```
Comment (jan-gajdowicz) [inline pages/kategoria/_category/_subcategory/index.vue:44]: Plase restore

File: models/category.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import API from '~/middleware/api.js'
+
+export const getProductsFromCategory = async  (category, page = 0, limit = 20) => {
```
Comment (jan-gajdowicz) [inline models/category.js:3]: The limit should be 24

**Review by jan-gajdowicz — APPROVED** [review]
GJ! :dancers:

File: models/category.js (hunk 3 above)
Comment (dloranc) [reply models/category.js]: Done.

**Pull request merged**